packages/plugin-splash-screen/src/legacy.ts

```diff
@@ -67,10 +67,17 @@ export async function ios(config: BuildConfig & CodePluginSplashScreen) {
     }
 
     /**
-     * There is no resource group by defaul so add in case another plugin has not
+     * There is no resource group by default so add in case another plugin has not
      * already created it. This would otherwise error out with vague message.
      */
-    project.addPbxGroup([], "Resources", '""');
+    if (!project.findPBXGroupKey({ name: "Resources" })) {
```

Comment:
probably worth adding this to v13 as it's a safer check and adds it to the main group